### ISSUE TITLE
Monthly Ticket for Quest Changes

### ISSUE THREAD

**Issue by chochem** [GTNewHorizons/GT-New-Horizons-Modpack#16658]
### Your GTNH Discord Username

chochem

### Your Pack Version

2.6.1

### Overview

The monthly ticket for quest changes. Well, maybe it is the _quarterly_ ticket for quest changes. A ticket for any large or small problems with any quests. Just post below. Of course, you can also make a dedicated ticket. In particular for more indepth suggestion or problems a dedicated one is better. This ticket is more focussed on small and clear problems and suggestions that don't really need any larger discussion.

The most up-to-date quest files can always be found here: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/tree/master/config/betterquesting/DefaultQuests
Depending on your version, you might also need to update BQ to use them.

Below are the remaining things from the last ticket (https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/15373):

### Quest Changes
- [ ] https://github.com/GTNewHorizons/GT5-Unofficial/pull/1146 reworked turbine materials, so quests related to turbine materials need to be checked to confirm if any changes are needed.
- [x] quest "Smaller Fusion Is Bigger" in "Powerful Nuclear Physics" should mention that compacts follow different overclocking rules, needing double the startup power and double the voltage instead of just the voltage.
- [x] some potential issues with AE2FC quests https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/15373#issuecomment-1979509272, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/15373#issuecomment-1979512989
- [x] add T9 planets to some quests https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/15373#issuecomment-2094254113
- [x] remove samarium oxide and mention of gangue from IV lanth quests
- [x] remove cynical comment at the end of arc furnace multi quest

### Wishlist for New Quests

- [x] quest regarding NEI's 2.6.0+ recipe group & linking mechanics?
- [x] quest regarding NEI's 2.7.0+ new search syntax options?
- [x] quests for new steam multis
- [ ] add quests for bastline (maybe not full detail but at least something?)
- [x] add Eye of Harmony quest + related stuff
- [ ] Add quest line or some tips on how to do Botania potions, specifically for Warp award flasks/alfglass/incense stick.
- [x] add quest for dark steel chest or even an entire ironchest progression.
- [x] quest for new PrAss tiers https://github.com/GTNewHorizons/GT5-Unofficial/pull/2698 (existing ones might also need changes)

Add more quests to the mobility tab:
- [ ] add quest for waystones
- [ ] add quest for ruby slippers
- [ ] add quest for infusion of otherwhere
- [ ] move spectre key quests here?
- [ ] add quest for arcane levitator?
- [ ] and to the rails the Botania Spectral rail??

### Potential Bigger Overhauls
- [x] the entire novice thaumaturgy tab
- [ ] the entire adept thaumaturgy tab
- [x] PRT quests (see https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/15373#issuecomment-2017335490
- [ ]  https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/840 for some reasons/details)

**Comment by GTNH-Afx237v7** [comment]
Something small but bothered me nonetheless:
„How to bee“-quest tree, the quest about the Genetic imprinter says that „it works 100% of the time“. I find this misleading - it does work if your bee doesn’t turn into genetic waste, because this machine also has the 30% degrade chance for pristine and 80% death chance for ignoble bees.
Suggestion: add an Information about a.m. degrade/death chances.

**Comment by Frikkle** [comment]
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/9324165/f65199fe-73af-49c8-8a5f-416df568e212)

The "Stainless Steel EBF processing" quest in MV is not dependent on the "MV Energy Hatch" quest. This made me think that it might be possible to overclock it with multiple LV Energy hatches, but correct me if I'm wrong, I think it's not.

In which case it might be more beneficial to make the quest dependent on the MV Energy Hatch quest, so it will become available after going through the motions needed to get to the energy hatch.

If there's anything I can do to help (make a PR or something) if this should be done, let me know! :)

**Comment by LooseEthics** [comment]
T4 rocket engine quest has the same description as the T3 quest.

**Comment by lilpaladin1** [comment]
Can we change the Adept Thaumaturgy quest "Secrets" so that it's instead hunting down 4 knights OR 4 clerics, so that way it avoids the problems from #2209 while also not being annoying for someone who has done the Nine Hells quest to get the Cleric Spawn Eggs only to find out that spawned Clerics don't spawn Knights?

**Comment by Imagine0123** [comment]
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/127961339/93383305-f09e-4ff5-b124-82a2b7c5f4d3)
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/127961339/8dd10b22-a64b-4e10-a61a-0bf447fe702f)

The quest for the Stocking input buses which are obtainable when you get AE2 is locked behind the quest for the UV-Tier stocking input hatches, feels like that's backwards

**Comment by chochem** [comment]
wasnt that fixed a while back?

**Comment by cargocats** [comment]
Typo in the word "durability" for the quest "Repairing Your Hazmat Suit"

![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/37312389/0e71cb55-2976-4d34-92cc-03955b9f048a)

**Comment by Baerschen** [comment]
We should adjust the me output hatch and bus quests for sampsas upcoming nerf. They should honestly get their own quest explaining the mechanichs that were shown in upcoming features channel. Will prob do this myself when its added to the nighlies just leaving this comment as a reminder.

**Comment by Doma1269** [comment]
![Screenshot 2024-07-22 214033](https://github.com/user-attachments/assets/76be6af6-dbb8-4340-8cd3-2ff0f1dba6ad)
1. Spelling mistake in LV microprocessor in HV
![image](https://github.com/user-attachments/assets/382f5081-1024-49ae-95c0-fdb852917504)
2. Small coils are locked behind Integrated processors, CPU`s and an MV circuit assembler. You can make them with the current tools you have (steel, copper and polyethylene)

**Comment by OlliedeLeeuw** [comment]
>     2. Small coils are locked behind Integrated processors, CPU`s and an MV circuit assembler. You can make them with the current tools you have (steel, copper and polyethylene)

Decided against changing this, while yes it is craftable earlier there is no reason to as the small coils are only used in the next quest (and in iv mainframe later) and for the purpose of keeping the quest flow intact.

**Comment by fw4hre0xxq** [comment]
Would it be possible to add a bypass for the HV Sifter quest to the GT++ Sifter Multiblock quest? I just realized I have to go back and make an HV Sifter, when I've already made several EV sifters. If possible a bypass when you hold a higher level single block sifter would be what I'm after.

**Comment by FourIsTheNumber** [comment]
> Would it be possible to add a bypass for the HV Sifter quest to the GT++ Sifter Multiblock quest? I just realized I have to go back and make an HV Sifter, when I've already made several EV sifters. If possible a bypass when you hold a higher level single block sifter would be what I'm after.

Isn't HV sifter part of the recipe for Large Sifter Controller?

**Comment by OlliedeLeeuw** [comment]
> Isn't HV sifter part of the recipe for Large Sifter Controller?

yeah it is, and the quest for GT++ sifter is also locked behind HV singleblock quest so i dont see the problem.

**Comment by qunYp3dZSyLjoEoEg** [comment]
Make the "The Transcontinental Railroad - Electric"-quest dependent on the Assembler quest in LV, as you can craft them without an EBF. 
Stone Railbed is easily available in the crafting table and better in the Assembler, and the electric Rail you can craft in the Assembler with just standard-rail and either copper or gold wire if you don't want to use any other machines. And the electric rails needed for the elctric tracks need an assembler to craft.

**Comment by chochem** [comment]
- [ ] move asoc quest to IV as it is needed for the precise assembler (which is now newly in IV)

**Comment by Baerschen** [comment]
If this https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/915#issue-2442407248 gets merged the quests for the fluid processor will need to be updated again

**Comment by dgealow** [comment]
"LOX and bagels" (explaining how to fill galacticraft O2 tanks with GT-derived oxygen) should have "Oxygen tanks" as a prereq and should *not* have "gas 'er up

**Comment by JL2210** [comment]
Remove the stone age quests pertaining to reloading the questbook as that's done automatically now. Command blocks and stick can be removed too

**Comment by Doma1269** [comment]
hv semifluid gens are locked behind a vacuum freezer
![image](https://github.com/user-attachments/assets/e3b4a3b8-b113-4850-9d5f-005151ae2870)

**Comment by SparkedTheorem** [comment]
![image](https://github.com/user-attachments/assets/f342a966-3f37-4a0b-a01d-db6172b91fab)
![image](https://github.com/user-attachments/assets/8d88491c-3a35-4b9b-b834-e52f4067e260)

- [ ] EV does not have a quest for its respective tier's mixer, which is inconsistent with the tier after it and the tiers before it.

**Comment by OneEyeMaker** [comment]
Quests for Logistics Pipes should be adjusted since this mod has been downgraded (https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/904).

**Comment by Atlantis102** [comment]
> Quests for Logistics Pipes should be adjusted since this mod has been downgraded ([GTNewHorizons/NewHorizonsCoreMod#904](https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/904)).

**use google translate**
provider module  MV ---> LV
request pipe MV ---> LV
fluid container supplier  MV ---> LV
upgrade module upgrade  MV ---> LV
crafting byproduct upgrade  MV ---> LV
fluid satellite pipe HV ---> MV

Lootbags for these pipes/modules higher than their crafting level

**Comment by SparkedTheorem** [comment]
- [ ] Tungstate Processing Part 1 mentions that processing is done so that smelting Tungsten doesn't take forever, this is presumably outdated information and now reads as misinformation to the player

**Comment by Ruling-0** [comment]
> * [ ]  Tungstate Processing Part 1 mentions that processing is done so that smelting Tungsten doesn't take forever, this is presumably outdated information and now reads as misinformation to the player

It elaborates in "Just Enough Tungstensteel for Your Rocket" that the tungstate > hot tungsten saves time over the pure tungsten recipe. This is true - it's 40% of the time per ingot compared to directly smelting the tungsten dust, which makes a big difference when the base time is 500s EV. This elaboration could probably be included directly with the line in question.

**Comment by ZaykieT** [comment]
hope im doing this right not too familiar with GH, the cleanroom quest has a letter where a space is supposed to be.
![image](https://github.com/user-attachments/assets/97653acc-a2df-48bb-8476-498ad1dca1c9)

**Comment by chochem** [comment]
> hope im doing this right not too familiar with GH, the cleanroom quest has a letter where a space is supposed to be. ![image](https://private-user-images.githubusercontent.com/108432213/356835935-97653acc-a2df-48bb-8476-498ad1dca1c9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzMjA2NDcsIm5iZiI6MTcyMzMyMDM0NywicGF0aCI6Ii8xMDg0MzIyMTMvMzU2ODM1OTM1LTk3NjUzYWNjLWEyZGYtNDhiYi04NDc2LTQ5OGFkMWRjYTFjOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxMFQyMDA1NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMWMxYTg3ZmY3NjBmZTJjZmE2ZTgxNjY2ZmUwNjJiMTFlYjg3OTRmZjRhOTE3MDRhNjY2OTU4MzE2MjY1NzNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.OlY2L9urC3oVFrmNWP0pgX1_2V9Eft6j79Vc4Dr47jw)

was fixed a while back

**Comment by Baerschen** [comment]
I think the quest for the Thermal Boiler is outdated. It got reworked and I find the description of the quest to be misleading as someone pointed out in discord.

**Comment by SasB18** [comment]
Expand the Coins coins coins chapter with some more purchasable item. For example gardener coin is added to the pack but it can't be used for anything.

**Comment by Baerschen** [comment]
> Expand the Coins coins coins chapter with some more purchasable item. For example gardener coin is added to the pack but it can't be used for anything.

This was already done. Gardener coins can now be used to purchase glimmering mushrooms if you have trouble finding them.

**Comment by FireWarrior2** [comment]
Add quests for new AE cards:
- [ ] Advanced Blocking Card
- [ ] Locking Card
- [ ] Pattern Refiller Card
- [ ] Sticky Card
- [ ] Hyper-Acceleration Card

**Comment by FireWarrior2** [comment]
- [ ] ZPM solar quest wants UEV circuit quest (Master of Circuits) but only UHV circuit is required.
![2024-08-18_15 10 31](https://github.com/user-attachments/assets/3ea2aa17-9a09-4358-b336-6f606618416e)

**Comment by LazyFleshWasTaken** [comment]
Add quests for Alternative boat progression:

* [ ] Greatwood boat
* [ ] Thaumium boat

Greatwood boat needs barely any thaum progress, it's slightly faster than a vanilla boat, and doesn't break on impact.
Thaumium boat is a bit farther in, but it can swim in lava.

**Comment by dgealow** [comment]
- [ ] "Shake that booty..." (the large sifter multiblock quest) tells you to make fewer industrial sieve casings than you actually need

The "maximum" possible number of casings in the multi structure is 25 + 16 + 16 = 57. Subtracting out the controller, maintenance hatch, and muffler; two energy hatches; one input and one output bus; leaves 50. If NEI and the tooltip are correct that you need fluid I/O as well, that's 48. But QB only tells you to make 47 casings.

Also, if the fluid hatches are necessary somehow, this quest should explain what the heck they're for.

**Comment by OmdaCZ** [comment]
![image](https://github.com/user-attachments/assets/8bf82997-6a06-4d2e-9264-91e0ef73622b)
pH neutralization quest has wrong info about how much faster t4 water is over t3, should be 100% faster, not just 50% (nightly 602)

**Comment by dgealow** [comment]
EV circuit assembler quest is gated by making the emitter *and* sensor, even though it actually only needs the emitter

**Comment by dgealow** [comment]
The quests for tier II or III boiler multis should probably give an auto-complete for the lower-tier boiler multi quests.

**Comment by LazyFleshWasTaken** [comment]
In the questbook, crops tab, there are a handful of rewards for crop bags. For a good portion of crops that require base blocks, usually, the reward has a part of that base block in itself, Flawless emerald in emerald, 4 diamonds in diareed, coal in withered, thorium ore for god of thunder, pitcheblencde for reactoria, glowstone bocks for glow crops, knightmetal block for knightly oreberry, and thaumium for magic metal berry.

With this established, this consistency should be kept with Garnydia, sapphrium, and the stonelilies. 8 out of 11 quests have the base block or parts of it, the other 3 don't.

**Comment by Dyonovan** [comment]
With the addition of the new steam multi's that take us into MV when upgraded, it would be nice to remove the requirements for the single block versions that lock progression in the LV/MV tabs.

ie: The LV centrifuge is required in the quest log for other things
![image](https://github.com/user-attachments/assets/8599c055-1e6c-46dd-a7e7-23243504acb6)

**Comment by dgealow** [comment]
- [ ] "Appropriate Power" quest in EV should point you towards the entire "Powerful Nuclear Physics" tab in the questbook, rather than just telling you to go look it up on the internet

**Comment by dgealow** [comment]
- [ ] Specify in the farming station quest that you need to plant the CROP rather than the seed for Pam's crops

**Comment by SparkedTheorem** [comment]
- [ ] Really (Really!) Dark Magic, the quest for Crimson Rites, should have its requirements untied as killing Crimson Knights is not mandatory to obtain them, you can either craft them or kidnap a Crimson Cultist and EEC them

**Comment by dgealow** [comment]
- [ ] Quests for higher-tier AE2 components (e.g. 64k storage components) debatably shouldn't have a strict dependency for the worse tiers

**Comment by Dyonovan** [comment]
On the Multiblock Goals page, the quest Digging Deep for Water (ID: jreOZVXRQa2sGXViV7-3Gg==) Is not able to be completed as it says there is a pre-requisite but its not showing what it is. I have completed the quest that forms the line to it.

![image](https://github.com/user-attachments/assets/dea5c6e7-77c8-4288-a136-3a4f722069af)

**Comment by Baerschen** [comment]
> On the Multiblock Goals page, the quest Digging Deep for Water (ID: jreOZVXRQa2sGXViV7-3Gg==) Is not able to be completed as it says there is a pre-requisite but its not showing what it is. I have completed the quest that forms the line to it.
> 
> ![image](https://private-user-images.githubusercontent.com/123577/365746788-dea5c6e7-77c8-4288-a136-3a4f722069af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU5MDI5MTgsIm5iZiI6MTcyNTkwMjYxOCwicGF0aCI6Ii8xMjM1NzcvMzY1NzQ2Nzg4LWRlYTVjNmU3LTc3YzgtNDI4OC1hMTM2LTNhNGY3MjIwNjlhZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwOVQxNzIzMzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MjFmN2ZiMTYyMGQzODhiMjdlYmRjNzFmYzkyNmNkYWVhY2E0YWJhOTc2ZDU1Yjg1MTVhODJkY2Q1MGFmNjY1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.P39c11YLR4_A-7EsZERgdxHgqmani2ZHZdPVrQae3qc)

Already fixed in newer nightlies

**Comment by SparkedTheorem** [comment]
- [ ] Mana Diamonds quest should either also accept Shake that Booty or not have any dependencies outside of Mana pool quest since you can use regular diamonds

**Comment by Phiapsi** [comment]
![grafik](https://github.com/user-attachments/assets/c11af608-ebb5-4a12-a870-bc5b5cecd4c9)

Should be "Alternative" instead of "Alterative"

- [ ] Typo of Telepad Quest in Getting Around

**Comment by Dyonovan** [comment]
The quest High-Speed Charcoal to make the Advanced Coke Oven is very misleading. The Advanced Coke Oven doesn't even make charcoal. Maybe rename the quest to High-Speed Coal Coke?

**Comment by gasgsm** [comment]
Quest "Applied Greg?" - add mention that Output Hatch (ME) can handle all multiblock outputs. Until now I was using them as ordinary output hatches - if recipe has 3 fluid outputs my LCR would have 3 me output hatches.

**Comment by JL2210** [comment]
Can we combine 'Another Brick in the Wall' with 'Coke Oven Brick' and make the bricks themselves an optional task? Every time I play this part I always smelt and craft the bricks first before realizing I need to have 104 of the cooked and uncooked ones in my inventory

**Comment by chochem** [comment]
- [ ] enchanted cloth discount is 5%, not 6%.

**Comment by SparkedTheorem** [comment]
- [ ] The Runes quest in the Botania section should only require you to make 1 of each rune. For most runes, you only need like one, as crafting the more complicated runes will return the runes used in crafting and I sue as hell do not need more than 1 of the complicated runes for basic progression.

**Comment by SasB18** [comment]
![2024-09-22_10 19 54](https://github.com/user-attachments/assets/2cc52d8b-b94e-45c4-ba8d-0df8bf572b21)
Typo in this quest: In order to make HV circuits, you need HV circuit

I found another typo, but I can't find it right now

**Comment by chochem** [comment]
> In order to make HV circuits, you need HV circuit

according to the picture it does not say that. What is the typo?

**Comment by SasB18** [comment]
> according to the picture it does not say that. What is the typo?

To me it just seemed like a typo, shouldn't it be more like: In order to make HV circuits, you need MV circuits. Maybe it's not a typo, but it didn't make much sense to me first

**Comment by chochem** [comment]
just say which word you mean in which sentence

**Comment by SasB18** [comment]
First sentence, 9th word

**Comment by chochem** [comment]
yea thats definetely correct.

**Comment by Yamnasm** [comment]
![image](https://github.com/user-attachments/assets/a12732bf-f4e8-4212-9e6d-95276a7835bd)

Typo found for quest "Handle Resources like a Bo$$" (LV-tier)
Typo highlighted in red. Should be "filter".

**Comment by dgealow** [comment]
> > according to the picture it does not say that. What is the typo?
> 
> To me it just seemed like a typo, shouldn't it be more like: In order to make HV circuits, you need MV circuits. Maybe it's not a typo, but it didn't make much sense to me first

You're slightly misreading--it says, in order to make HV ***machines***, you need HV circuits.

**Comment by purchan** [comment]
Quest "Rutile" in "Tier 3 - HV"
![rutile](https://github.com/user-attachments/assets/4d541103-b7c7-4864-a52b-24174f82c768)

The current quest description is not clear enough. If one uses single-block chemical reactor and carbon dust (indicated by optional task) as input, they can't recycle CO to get all the carbon back. To get CO, one must use carbon cell as input.
![crnco](https://github.com/user-attachments/assets/389642be-7c45-4765-9391-59f9b2088ed8)
![crco](https://github.com/user-attachments/assets/aa4b0790-c5e1-441c-91ff-2844e768f5c5)
![lcr](https://github.com/user-attachments/assets/b73c2142-e8ad-4adf-ae73-a294d61825a9)

Modified quest description:
```
You've gotten some Rutile as a byproduct by now, right? Or EBF'd some Ilmenite?

Rutile, Chlorine and Carbon makes Titaniumtetrachloride, and some CO. You can recycle CO to get all the Carbon back.
(If you're using a single-block chemical reactor, use carbon cells instead of carbon dust to obtain CO for recycling.)

Combine TiCl4 with Magnesium in an HV EBF to make Hot Titanium ingot and Magnesiumchloride. You can recyle MgCl2 as well to get all the Magnesium and Chlorine back using some Sodium.
```

**Comment by chochem** [comment]
it should just say to use LCR

**Comment by purchan** [comment]
But there is an option to use single-block chemical reactor and still get back all the carbon. I think it is worth mentioning. Canning carbon dust into carbon cell is almost zero-cost, requiring a canner.

**Comment by leumasme** [comment]
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/blob/bbfe04e51e389d783bd7b757e8b79a54cbdac96f/config/betterquesting/DefaultQuests/Quests/FishingFarmingCo-AAAAAAAAAAAAAAAAAAAACg%3D%3D/BetterDirt-AAAAAAAAAAAAAAAAAAAHAQ%3D%3D.json#L34
![image](https://github.com/user-attachments/assets/6723c45f-e394-4556-93b1-86388d7b0f81)
I think this part can be removed as it should've been caused by ic2 crops and thus fixed in https://github.com/GTNewHorizons/Hodgepodge/pull/425

**Comment by chochem** [comment]
- [ ] we might need some quest adjustments due to vein name changes https://github.com/GTNewHorizons/GT5-Unofficial/pull/3290

**Comment by FireWarrior2** [comment]
- [ ] Probably QFT quest needs Infinity coil quest requirement because it is used in QFT coil casing craft
![2024-09-30_17 29 08](https://github.com/user-attachments/assets/894d210a-96ba-44d3-bb82-20f78ca6ab1b)
![2024-09-30_17 31 09](https://github.com/user-attachments/assets/d4150d99-b057-4f0f-9199-b70d8fdad0f9)

**Comment by SasB18** [comment]
Found a typo in the Cleanroom quest, this time for real. Line 7, it says "top center,fFilter machine...". Should be "top center, Filter machine..."
![2024-10-06_15 54 01](https://github.com/user-attachments/assets/79cbb0e2-ef19-4c04-a84f-032fd62800f5)

**Comment by chochem** [comment]
pretty sure that was fixed. I guess you are on stable which has been a while :P

**Comment by tommmmmmmmm** [comment]
plasma mixer no longer requires UMV circuit, i think it can be moved next to the dtpf quest.

**Comment by HoleFish** [comment]
LCR can overclock beyond 1 tick now.
![image](https://github.com/user-attachments/assets/911f4552-51bc-447f-aa39-5b1e27994d95)

**Comment by Baerschen** [comment]
- [ ] Quests for the new Item Holders introduced in 2.7 (will probably do that one myself once i get my hand on beta-3 and see where to best put it based on recipe)
- [ ] Quests for ravens new jukebox. They would fit best in Building better bases and could reward a pair of Headphones if you get yourself an electrical jukebox
- [ ] Some bees that are very worthwile that could use quests (sunnarium, energium and laoptron)
- [ ] Add info to the cactus and sugar charcoal quests that they can now be compressed to last longer (new in 2.7)
- [ ] Adjust the quest in the MV tab that explains options for power since it is now outdated (e.g. for now there is no magic route since LEG got removed)

**Comment by tommmmmmmmm** [comment]
UMV components quest requires field generator, which then has its own quest gated by shirabon/uxv circuit.
can we get the field gen (and maybe also emitter and sensor) out of the other components so i don't have to make it before the 43 component casings and also not have it being asked by two consecutive quests?

![image](https://github.com/user-attachments/assets/57bfd0c2-0a83-45f3-902e-5d8768822437)
![image](https://github.com/user-attachments/assets/60cab709-c74b-4dc3-9b39-b7ec5bfad137)

**Comment by tommmmmmmmm** [comment]
metastable oganesson is an UMV quest, but it was (implicitly?) required in UIV as ingredient for the dimensionally shifted superfluid that goes into UIV motors

**Comment by Demiu** [comment]
There is no quest for varja, I think it's last gate is IV engraver for engraved energy chip

https://github.com/GTNewHorizons/ArchitectureCraft/pull/19 added a neato glowbrush to turn existing AC blocks glowing, would be a nice subquest to the architecturecraft sawbench

**Comment by chochem** [comment]
![image](https://github.com/user-attachments/assets/c6db82d9-646b-434d-9a6a-4fc7138678e2)
whats this then :P

**Comment by Ethryan** [comment]
Some info about the Akashic Tome and how to use it might be a good idea.

**Comment by SasB18** [comment]
Suggestion: Change this choice quest. Even the questbook suggests that we shouldn't choose the loot, so why not just give the player the personal dim in the moon dungeon quest as a reward?
![2024-10-20_12 18 33](https://github.com/user-attachments/assets/63f72ac3-dcc6-498d-91ee-ce7bdb517fbd)

**Comment by Miklebe** [comment]
![image](https://github.com/user-attachments/assets/4cd8394e-7356-4bff-88cd-c336663a0e12)
The apiary chest quest asks for GT version of the crate, while the craft only uses forestry's version.
![image](https://github.com/user-attachments/assets/2727d383-7e11-4824-a275-a9ac7c587217)

**Comment by FireWarrior2** [comment]
- [ ] Infinity spray can not given after completing platline.
If I understood quest correct, it should be given. Possibly because Platline skip tasks were done, not main chain ones.
![image](https://github.com/user-attachments/assets/66a9bcc5-5d7c-42ef-8f40-723db88c3ae7)

**Comment by chochem** [comment]
"was not given"? what does that mean?

**Comment by chochem** [comment]
lol. no.

**Comment by FourIsTheNumber** [comment]
> If I understand this quest correct, spray should be given as a reward after completing platline.
No. It's saying that after completing platline you are now able to craft it, since it is made of platline materials.

**Comment by FireWarrior2** [comment]
> > If I understand this quest correct, spray should be given as a reward after completing platline.
> 
> No. It's saying that after completing platline you are now able to craft it, since it is made of platline materials.

This is not a reward then)) This word usage can confuse.
It just become possible to use.

**Comment by chochem** [comment]
it most certainly is a reward for finishing platline

**Comment by FourIsTheNumber** [comment]
It even goes more in detail here. "Naturally, you'll have to craft the infinite spray can yourself..." etc.

**Comment by Miklebe** [comment]
![image](https://github.com/user-attachments/assets/cdd38ba2-3f1b-4224-96d6-bba324d2532b)
Not a big one, but having controller both in optional and retrieval tasks seems unnecessary?

**Comment by FireWarrior2** [comment]
- [ ] This should be removed with cost rebalance
![image](https://github.com/user-attachments/assets/e7bd1de2-a728-4201-9c01-3eaf870c206f)
